sys/cheri/cheri_sysctl.c

```diff
@@ -84,3 +90,39 @@ u_int	security_cheri_bound_legacy_capabilities;
 SYSCTL_INT(_security_cheri, OID_AUTO, bound_legacy_capabilities,
     CTLFLAG_RWTUN, &security_cheri_bound_legacy_capabilities, 0,
     "Set bounds on userspace capabilities created by legacy ABIs.");
+
+static void
+measure_cloadtags_stride(void *dummy __unused)
+{
+	void * __capability *buf;
+	uint64_t tags;
+	u_int i;
+
+	/*
+	 * Malloc a buffer as allocating an aligned page on the stack
+	 * risks overflowing the stack.
+	 */
+	buf = malloc_aligned(sizeof(*buf) * 64, sizeof(void * __capability),
```

Comment:
Alignment must equal size, since CLoadTags faults for addresses that aren't `caps_per_cache_line * cap_size`-aligned (where `caps_per_cache_line` should really be abstracted as `caps_per_cloadtags`).

Reply:
Humm, I guess I can see why that additional requirement is present for CLoadTags, and in practice it's always done on pages so far, so it's not a burden to deal with.